src/components/addnewbook.css

```diff
@@ -0,0 +1,41 @@
+.addcontainer {
+  margin-top: 3rem;
+  margin-bottom: 10rem;
+}
+
+.addcontainer h1 {
```

Comment:
- [ ] Kindly use the same font and color as the one given in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
Please always keep in mind that following a given design to be EXACTLY as it is, is a very important skill for a software engineer, and here's another chance for you to practice it 😄 🙏 

_**Kindly follow the given design in all aspects, colors, font-size, font-weight, font-family, and spacings (margin and padding)**_

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/132088695-09724600-9315-436e-8a62-27575a1d36be.png)

>Please apply the changes required on this point all over the whole page!